src/UBAFeeCalculator/UBAFeeSpokeCalculator.ts

```diff
@@ -167,61 +172,82 @@ export default class UBAFeeSpokeCalculator {
     return this.config.getBalanceTriggerThreshold(this.chainId, this.symbol);
   }
 
+  /**
+   * A convenience method for resolving the event fee for the spoke and the given symbol and a given flow type
+   * @param amount The amount of tokens to simulate
+   * @param flowType The flow type to simulate
+   * @param flowRange The range of the flow to simulate the event for. Defaults to undefined to simulate the event for the entire flow.
+   * @returns The event fee for the spoke and the given symbol and a given flow type
+   */
+  protected getEventFee(amount: BigNumber, flowType: "inflow" | "outflow", flowRange?: UBAFlowRange): FlowFee {
```

Comment:
Can you use the sign of `amount` to infer the `flowType`? Just seeing if there's a way to remove the second param

Reply:
That should be fine. But would it ever be possible (even in a contrived case to rescue funds or something similar) to want to encode the ability to handle an inverse deposit/refund event?

Reply:
hmmm thats an interestingpoint. I can't imagine this case, wouldn't you just call `getEventFee(-amount)` (and negate the sign) to compute the hypothetical reverse event?

Reply:
One option would be to pass in the full `Flow` object, and then use the `isInflow` and `isOutflow` predicates to determine which path to take. Presumably that's happening outside of this scope, per the fact that there are separate methods for inflow and outflow.